kotlin/android/AmpliApp/app/src/main/java/com/example/ampliapp/App.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.ampliapp
+
+import android.app.Application
+import com.amplitude.ampli.*
+
+class App : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        val amplitudeApiKey: String? = null // Set your Amplitude API key
+        ampli.load(
+            LoadOptions(
+                androidContext = this,
+                environment = Ampli.Environment.DEVELOPMENT,
+                client = LoadClientOptions(apiKey = amplitudeApiKey)
+            )
+        )
+    }
```

Comment:
Can you please add (commented out) examples of other Ampli usages like this - https://github.com/amplitude/ampli-examples/blob/main/browser/typescript/react-app/src/App.tsx#L17-L52

Reply:
Added. I've changed `load` signature from `load(options)` to `load(appContext, options?)` to simplify default usage